### PR TITLE
import data.monoid

### PR DESCRIPTION
**Pull request by KevinLe563**
Imported Data.Monoid to resolve "Variable not in scope: (<>)" error during cabal install

### CODE REVIEW

**Comment by igrep** [comment]
The error is caused because your GHC is older than GHC 8.4 (base-4.11.0). Please update if you can.

@krdlab Which version do you support old GHCs by?

**Comment by krdlab** [comment]
@KevinLe563
I'm sorry for replying late.

@igrep
Thank you for your helpful comment.

> Which version do you support old GHCs by?

I suppose that `oidc-client-0.3.0.1` can compile with an old GHC.

**Comment by krdlab** [comment]
Um...it may be better to import `<>` explicitly...

File: src/Web/OIDC/Client/Discovery.hs [hunk 1]
```diff
@@ -19,6 +19,7 @@ module Web.OIDC.Client.Discovery
     , generateDiscoveryUrl
     ) where
 
+import           Data.Monoid
```
Comment (krdlab) [inline src/Web/OIDC/Client/Discovery.hs]:
```suggestion
import           Data.Monoid                        ((<>))
```

File: src/Web/OIDC/Client/Tokens.hs [hunk 2]
```diff
@@ -14,6 +14,7 @@ module Web.OIDC.Client.Tokens
     )
 where
 
+import           Data.Monoid
```
Comment (krdlab) [inline src/Web/OIDC/Client/Tokens.hs]:
```suggestion
import           Data.Monoid                        ((<>))
```

**Review by krdlab — COMMENTED** [review]
Thanks for the PR. Would you modify it a little bit?

**Review by krdlab — APPROVED** [review]
Thank you

**Pull request merged**